netaddr_test.go

```diff
@@ -281,6 +340,13 @@ func mustIP(s string) IP {
 	return ip
 }
 
+func mustIPs(strs ...string) (res []IP) {
```

Comment:
Nit: I'm not sure the named return is necessary for a function with a single return.